mypyc/test-data/run-floats.test

```diff
@@ -20,3 +20,9 @@ def test_abs() -> None:
     assert abs(44324.732) == 44324.732
     assert abs(-23.4) == 23.4
     assert abs(-43.44e-4) == 43.44e-4
+
+[case testFloatMin]
+def test_float_min() -> None:
```

Comment:
I think merging these test cases into one might be better.

Reply:
Done